package.json

```diff
@@ -50,60 +50,60 @@
   },
   "dependencies": {
-    "ember-cli-babel": "^7.26.10",
+    "ember-cli-babel": "^7.26.11",
     "ember-validators": "^4.1.1"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.6.0",
-    "@embroider/test-setup": "^0.48.1",
-    "@glimmer/component": "^1.0.4",
-    "@glimmer/tracking": "^1.0.4",
+    "@ember/test-helpers": "^2.7.0",
+    "@embroider/test-setup": "^1.6.0",
+    "@fortawesome/ember-fontawesome": "^1.0.0",
+    "@fortawesome/free-brands-svg-icons": "^6.4.0",
+    "@fortawesome/free-solid-svg-icons": "^6.4.0",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
+    "@release-it-plugins/lerna-changelog": "^5.0.0",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^2.7.0",
     "ember-auto-import": "^2.4.2",
-    "ember-cli": "~4.0.1",
+    "ember-cli": "~4.4.1",
     "ember-cli-app-version": "^5.0.0",
-    "ember-cli-autoprefixer": "^0.7.0",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-github-pages": "^0.2.0",
+    "ember-cli-autoprefixer": "^2.0.0",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-github-pages": "^0.2.2",
     "ember-cli-htmlbars": "^6.0.1",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-moment-shim": "^3.3.3",
-    "ember-cli-sass": "^8.0.1",
+    "ember-cli-moment-shim": "^3.8.0",
+    "ember-cli-sass": "^11.0.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-cli-yuidoc": "^0.8.8",
     "ember-code-snippet": "^3.0.0",
-    "ember-data": "~3.28.0",
+    "ember-data": "~4.4.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.1",
-    "ember-font-awesome": "^3.0.5",
     "ember-load-initializers": "^2.1.2",
-    "ember-prism": "^0.12.0",
+    "ember-prism": "^0.13.0",
     "ember-qunit": "^5.1.5",
     "ember-resolver": "^8.0.3",
-    "ember-source": "~4.0.1",
+    "ember-source": "~4.4.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^3.15.0",
-    "ember-truth-helpers": "^3.0.0",
+    "ember-template-lint": "^4.8.0",
     "ember-try": "^2.0.0",
     "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.8",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-ember": "^10.6.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-qunit": "^7.2.0",
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.5.1",
-    "qunit": "^2.17.2",
+    "prettier": "^2.6.2",
+    "qunit": "^2.19.1",
     "qunit-dom": "^2.0.0",
-    "release-it": "^14.2.1",
-    "release-it-lerna-changelog": "^3.1.0",
-    "sass": "^1.15.3",
-    "webpack": "^5.65.0",
-    "yuidoc-ember-theme": "^2.0.1"
+    "release-it": "^15.10.3",
+    "sass": "^1.62.1",
+    "webpack": "^5.72.1"
   },
   "engines": {
     "node": "12.* || 14.* || >= 16"
```

Comment:
Was node 12 dropped? Noticed you bumped above

Reply:
No, that will happen in ember-cli `v4.8`. I think it was a `devDependency` that needed `v12` already.

Reply:
If you put the breaking label because of node v12 I don't think we need it. But the next PR will be breaking. I'll highlight that in the PR.